.circleci/config.yml

```diff
@@ -165,6 +165,11 @@ jobs:
             tar --exclude="node_modules" -cjf pyodide-build.tar.gz dist
             tar cjf build-logs.tar.gz  packages/build-logs
 
+      - run:
+          name: Clean up package source files
+          command: |
+            cd packages && find **/build ! -name '.packaged' -type f -exec rm -f {} +
```

Comment:
What does the ! do?

Reply:
It means "not". It is there in order to exclude `.packaged` file from remove targets.